packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/__tests__/duration-mobile.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import TraderProviders from '../../../../../../../trader-providers';
+import DurationMobile from '../duration-mobile';
+import { mockStore } from '@deriv/stores';
+
+jest.mock('@deriv/components', () => {
+    return {
+        ...jest.requireActual('@deriv/components'),
+        Tabs: jest.fn(({ onTabItemClick, children }) => <div onClick={() => onTabItemClick(0)}>{children}</div>),
```

Comment:
SonarCloud is complaining on this line because of onClick on div. I got rid of this errors by mocking in another way:
`Tabs: jest.fn(({ onTabItemClick, children }) => <div>{children}<button onClick={() => onTabItemClick(0)}>Some text</button></div>),`